examples_server/examples/data_table/server_side/data_table.js

```diff
@@ -1,9 +1,9 @@
 Backdraft.app("TableExample", {
 
-  plugins : [ "DataTable"],
+  plugins: [ "DataTable"],
```

Comment:
is there a linter we can add to check for this? Looks like it missed an unecessary leading space  in the array

Reply:
yep there is a linter for this: `"standard/array-bracket-even-spacing": ["error", "either"],`. We would have seen it if hound ran on this PR...

Reply:
Why isn't Hound running on this PR? :-)

Reply:
It looks like it finally ran yesterday after I pushed up another commit..